app/main/views/sign_in.py

```diff
@@ -213,6 +214,11 @@ def sign_in():  # pragma: no cover
     ):
         return email_verify_template
 
+    redirect_url = request.args.get("next")
+
+    if os.getenv("NOTIFY_E2E_TEST_EMAIL"):
+        return _handle_e2e_tests(redirect_url)
+
```

Comment:
Double checking again with @terrazoon on this - should this have moved like, or does it need to go back to where it originally was?

Reply:
we want to skip all login.gov processing, so I moved the check for the email to the top of the method.  It can be verified easily if anything breaks:

1. Do e2e tests still pass?
2. Can you still log in?

Reply:
@alexjanousekGSA were things not working with this code at the top of the method?  Or did that update somehow not roll into your other changes here with your last sync of `main` in this branch?

Reply:
Okay, somehow it didn't make its way back to the top of the method. I moved it back up there and tested it and it seems to all work fine!